DNN Platform/Library/Common/Utilities/StringExtensions.cs

```diff
@@ -65,6 +65,21 @@ public static string ReplaceIgnoreCase(this string source, string oldValue, stri
             }
 
             return Regex.Replace(source, Regex.Escape(oldValue), newValue, RegexOptions.IgnoreCase);
+        }
+
+        /// <summary>
+        /// Removes new line separators.
+        /// </summary>
+        /// <param name="source">The source.</param>
+        /// <returns>The string without new line separators.</returns>
+        public static string RemoveNewLineSeparators(this string source)
+        {
+            if (string.IsNullOrWhiteSpace(source))
+            {
+                return source;
+            }
+
+            return Regex.Replace(source, @"\t|\n|\r", string.Empty);

```

Comment:
If we're going to include tabs (`\t`) then let's not call the method `RemoveNewLineSeparators`.